etc/events.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!--
+  ~ @author Aligent Consulting Team
+  ~ @copyright Copyright (c) 2022 Aligent Consulting. (http://www.aligent.com.au)
+  -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
+    <event name="sales_order_save_commit_after">
+        <observer name="sendOrderSaveEvent" instance="Aligent\DefaultAsyncEvents\Observer\OrderSave"/>
+    </event>
+    <event name="sales_order_shipment_save_commit_after">
+        <observer name="sendShipmentSaveEvent" instance="Aligent\DefaultAsyncEvents\Observer\ShipmentSave"/>
+    </event>
+    <event name="customer_save_commit_after">
```

Comment:
I've found this event to be a bit unreliable. Fires off multiple times in several cases.

Reply:
I've noticed this also, but I'm not sure there's a great alternative.

Reply:
In most cases this was fine I believe. 

The biggest one was actually with a different one. An event observer on the `customer_address`.

with the GraphQL `updateCustomerAddress`, if the customer has N addresses and a default shipping and billing, the `customer_address_save_commit_after` fires off (2 + N) times.

Just something to keep in mind.